routes/api/contacts.js

```diff
@@ -5,18 +5,37 @@ import {
   listContacts,
   removeContact,
   updateContact,
+  updateContactFavoriteStatus,
 } from "../../models/contacts.js";
+import {
+  addContactSchema,
+  updateContactSchema,
+} from "../../validation/validationJoi.js";
 
 const router = express.Router();
 
 router.get("/", listContacts);
 
 router.get("/:contactId", getContactById);
 
-router.post("/", addContact);
+router.post("/", async (req, res, next) => {
+  const { error } = addContactSchema.validate(req.body);
+  if (error) {
+    return res.status(400).json({ message: error.details[0].message });
+  }
+  addContact(req, res, next);
+});
```

Comment:
Może warto całość przenieść do kontrolera? Żeby tu było jedna linijka? Do zastanowienia czy to nie ulepszy czytelności
```suggestion
router.post("/", addContact);
```